### ISSUE TITLE
Minimized HTML attributes

### ISSUE THREAD

**Issue by cviebrock** [LaravelCollective/html#44]
Currently, if you pass in a numerical-keyed attribute, it will assume the key and value are the same.  For example:

```php
echo Form::select('choices', $choices, [ 'class' => 'foo', 'multiple' , 'required'] );
```

This will render with overly verbose attributes which are redundant, especially especially in HTML5.

```html
<select name="choices" id="choices" class="foo" multiple="multiple" required="required">
...
</select>
```

Could the HtmlBuilder::attributeElement() method be modified so that numerically-keyed attributes just return a minimized (i.e. value-less) attribute? Maybe something like:

```php
protected function attributeElement($key, $value)
{
	// For numeric keys we will assume that the value is the key and the attribute is
	// to be minimized.  This will render attributes like "required" or "multiple" correctly, 
	// instead of using numerics.
	if (is_numeric($key)) return $value;

	if ( ! is_null($value)) return $key.'="'.e($value).'"';
}
```

This would then render as the smaller (and still valid):

```html
<select name="choices" id="choices" class="foo" multiple required>
...
</select>
```

**Comment by tshafer** [comment]
Can you make a pull request with a proposal on how this would be solved?

**Comment by adamgoose** [comment]
This issue is being addressed in #50.
